tsconfig.json

```diff
@@ -12,7 +12,7 @@
     // "disableReferencedProjectLoad": true,             /* Reduce the number of projects loaded automatically by TypeScript. */
 
     /* Language and Environment */
-    "target": "es2018" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
+    "target": "es2023" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
```

Comment:
I've bumped this to `es2023` as this should be fully supported in node22, which we are using. This was necessary to be able to use the `AggregateError` type in `isNetworkConnectivityError`